python/eups/tags.py

```diff
@@ -218,6 +218,7 @@ def load(self, group, file):
                           from configured location.
         """
 
+        fd = None
         try:
```

Comment:
This try block should be a `with open(file) as fd` and then the finally wouldn't be needed.

Reply:
True.  Is this worth changing (the code precedes with, of course)?

Reply:
It needs to be done at some point but doesn't need to be done today.